web-ng/src/app/services/observation/observation.service.ts

```diff
@@ -98,9 +98,7 @@ export class ObservationService {
       .getLayer(feature.layerId)!
```

Comment:
don't we need a check now user != ANONYMOUS_USER or are you planning to allow ANONYMOUS_USER to leave observation?

Reply:
As we currently have it configured, Firebase will refuse unauthenticated users, so the only time data can be submitted anonymously is to the local emulator setup, which is WAI. Please cmiiw!

Reply:
well i just wondered what the user would see it if happens?:P

Reply:
If the user is able to access this UI at all then there's something wrong, since we currently require authentication to see the basic map UI. But even if they can access the map UI without logging in, Firebase won't allow them to save data to the db, and an error will be written to the console. In general, we should do better than fail silently, but in this case reaching that point would be impossible.